addons/rose/addon/components/rose/pagination/index.hbs

```diff
@@ -4,6 +4,7 @@
 }}
 
 <Hds::Pagination::Numbered
+  data-test-pagination
```

Comment:
I am open for a less generic name for this. I opted for a generic name since we shouldn't have 2 of these on the same page but never say never. Thoughts?

Reply:
I think its a good name. I agree, we shouldn't have more than 1 pagination component per page 😉